src/config/networks/alfajores.ts

```diff
@@ -6,12 +6,11 @@ import {
   NetworkConfig,
   WALLETS,
 } from 'src/config/networks/network.d'
-import { ETHGASSTATION_API_KEY } from 'src/utils/constants'
 
 const baseConfig: EnvironmentSettings = {
```

Comment:
some of the urls here are certainly wrong. some i think are correct some i know are